pkg/cmd/kafka/create/create.go

```diff
@@ -809,12 +833,48 @@ func determineFlowFromQuota(opts *options) error {
 	case opts.hasLegacyQuota:
 		opts.useLegacyFlow = true
 		return nil
+	case opts.useTrialFlow:
+		opts.useLegacyFlow = true
+		return nil
 	default:
 		opts.useEnterpriseFlow = true
 		return nil
 	}
 }
 
+func doesUserHaveTrialInstances(opts *options) (bool, error) {
+	conn, err := opts.f.Connection()
+	if err != nil {
+		return false, err
+	}
+
+	api := conn.API()
+
+	cfg, err := opts.f.Config.Load()
+	if err != nil {
+		return false, err
+	}
+
+	userName, ok := token.GetUsername(cfg.AccessToken)
+	if !ok {
+		return false, opts.f.Localizer.MustLocalizeError("kafka.create.error.cannotFindUserDetails")
+	}
+
+	list, _, err := api.KafkaMgmt().GetKafkas(opts.f.Context).Page(strconv.Itoa(1)).Size(strconv.Itoa(99)).Execute()
+	if err != nil {
+		return false, err
+	}
```

Comment:
Another example of when having ONLY pagination creates pain, I am not sure what to do here to make sure we are always getting every Kafka in the org @dimakis any thoughts

Reply:
left a comment about what needs to be done from the API: https://github.com/redhat-developer/app-services-cli/pull/1840/files#r1146358149 then this call only becomes about filtering for a developer kafka owned by the user. If the list is empty then we are good. If the list is not empty then it'll be one element.